setup/pi/setup-teslausb

```diff
@@ -918,11 +918,14 @@ sed -i '/source \/root\/bin\/setup-teslausb/d' /root/.bashrc
 if ! grep -q TESLAUSB_TIP1 /root/.bashrc
 then
   cat >> /root/.bashrc <<- EOC
-	cat << TESLAUSB_TIP1
-	Run 'bin/setup-teslausb upgrade' to update to the latest version of teslausb,
-	or run 'bin/remountfs_rw' to allow writing to the root partition.
-
-	TESLAUSB_TIP1
+	if [ -n "\$PS1" ]
+	then
```

Comment:
Can you change this back to `if [ ...] ; then`, like it is below on line 938?
While the TeslaUSB scripts use the if-on-the-next-line style, this `if` is written to .bashrc, which uses the if-on-the-same-line style.